src/components/Forms/Textarea.jsx

```diff
@@ -37,6 +37,7 @@ const Textarea = ({
         name={name}
         placeholder={placeholder}
         value={value}
+        style={{ resize: "none", height: "50px" }}
```

Comment:
this can be done via tailwind, make the height based on characters if possible, but everything should be in tailwind